var/spack/repos/builtin/packages/qt/qt4-mac.patch

```diff
@@ -215,9 +214,23 @@ diff -r -u qt-everywhere-opensource-src-4.8.7.orig/mkspecs/common/g++-base.conf
  
  QMAKE_LINK       = $$QMAKE_CXX
  QMAKE_LINK_SHLIB = $$QMAKE_CXX
-diff -r -u qt-everywhere-opensource-src-4.8.7.orig/mkspecs/common/gcc-base-macx.conf qt-everywhere-opensource-src-4.8.7/mkspecs/common/gcc-base-macx.conf
---- qt-everywhere-opensource-src-4.8.7.orig/mkspecs/common/gcc-base-macx.conf	2015-05-07 10:14:42.000000000 -0400
-+++ qt-everywhere-opensource-src-4.8.7/mkspecs/common/gcc-base-macx.conf	2019-04-04 15:47:55.000000000 -0400
+diff -Naur a/mkspecs/common/g++-macx.conf b/mkspecs/common/g++-macx.conf
+--- a/mkspecs/common/g++-macx.conf	2015-05-07 09:14:42.000000000 -0500
++++ b/mkspecs/common/g++-macx.conf	2022-12-22 17:01:56.000000000 -0600
+@@ -16,8 +16,8 @@
+ 
+ QMAKE_LFLAGS_STATIC_LIB += -all_load
+ 
+-QMAKE_CFLAGS_X86_64 += -Xarch_x86_64 -mmacosx-version-min=10.5
+-QMAKE_CFLAGS_PPC_64 += -Xarch_ppc64 -mmacosx-version-min=10.5
++QMAKE_CFLAGS_X86_64 += -Xarch_x86_64 -mmacosx-version-min=$$QMAKE_MACOSX_DEPLOYMENT_TARGET
++QMAKE_CFLAGS_PPC_64 += -Xarch_ppc64 -mmacosx-version-min=$$QMAKE_MACOSX_DEPLOYMENT_TARGET
+ 
+ QMAKE_CXXFLAGS_X86_64         = $$QMAKE_CFLAGS_X86_64
+ QMAKE_CXXFLAGS_PPC_64         = $$QMAKE_CFLAGS_PPC_64
```

Comment:
This is the only real change in this PR. If you want, I can remove all the extraneous header changes too.